tests/search_disk_index.cpp

```diff
@@ -327,6 +352,15 @@ int main(int argc, char** argv) {
                        po::bool_switch()->default_value(false),
                        "Include full precision data in the index. Use only in "
                        "conjuction with compressed data on SSD.");
+    desc.add_options()(
+        "num_starting_points",
+        po::value<uint32_t>(&num_starting_points)->default_value(0),
+        "Number of starting points used for building index (defaults to 0");
```

Comment:
Why do we need start points used for building index in the search code?

Reply:
just expose one parameter to balance recall vs. latency. If you think we could just use all the start_points, then I could remove this

Reply:
Sorry, my question was why the num_Start_points in the index build needs to be passed again by user when index is invoked for search. WHy not save this info to the index on save and load it back up?

Reply:
users could configure to use a smaller num_start_points